Update request aggregation added (and some formatting)

fibrinogen opened the pull request:
Added configurable update request aggregation (e.g. collect updates for n minutes and then sent update request)

Commit:
Added configurable update request aggregation

Commit:
Set port back to 80

bgewehr commented:
Does your enhancement allow 0.5 minutes as agg time? I would prefer a setting in 30 seconds...

bgewehr commented:
I just compiled your Code with aggregation and i get told "Aggretation not active" in syslog (see typo...) although s0vz altered like you proposed but with 1 minute. Can you tell why?

bgewehr commented:
Found the error: parameter aggtime is wrong, should be aggmin!

bgewehr committed:
Update s0vz.c

Read aggmin param as float for lower agg times like 0.5 for 30s

bgewehr committed:
Update s0vz.c changed datatype of aggmin to float

bgewehr committed:
Update s0vz.c used double data type for aggmin

bgewehr committed:
Update s0vz.cfg changed aggrime to aggmin like expected in code of s0vz.c

bgewehr committed:
Update s0vz.cfg removed wrong ;

bgewehr committed:
Update s0vz.c 
changed back to int type for aggtime parameter aggmin renamed to aggtime takes seconds now
Removed * 60 for second usage
Removed typo aggregation
Changed log text to seconds

bgewehr committed:
Update s0vz.cfg
Changed from aggmin to aggtime
Changed from minutes to seconds

bgewehr committed:
Update s0vz.c corrected aggtime devlaration

fibrinogen committed:
Merge pull request #2 from bgewehr/patch-2

Update s0vz.c for second based aggregation implemented by bgewehr.

fibrinogen commented:
Sorry for not answering for along time. I had some private trouble...
